src/JackTrip.cpp

```diff
@@ -588,9 +588,23 @@ void JackTrip::completeConnection()
     for (auto& i : mProcessPluginsFromNetwork) {
         mAudioInterface->appendProcessPluginFromNetwork(i);
     }
+
+    VuMeter* outputMeterPlugin = new VuMeter(mNumAudioChansOut);
+    mAudioInterface->appendProcessPluginFromNetwork(outputMeterPlugin);
```

Comment:
I realize this work is still in progress, but why is this not done along with the other plugins in line 589? Same question for input.

Reply:
The other plugins are instantiated in `Effects.h` (see [here](https://github.com/jacktrip/jacktrip/blob/85fb914e2418fcfe86d23e2e26bdec92852c29b4/src/Effects.h#L77)), which I think is only used when parsing command-line options without a GUI. Or if there is a GUI, they're instantiated in `qjacktrip.cpp` because they are added based on whether certain boxes are checked (see [here](https://github.com/jacktrip/jacktrip/blob/85fb914e2418fcfe86d23e2e26bdec92852c29b4/src/gui/qjacktrip.cpp#L1333)). I think this plugin is a bit different from the others because it doesn't modify the audio signal (it's not really an "effect") and we don't include it based on whether certain boxes in classic mode are checked. 

Adding the VU plugin at `QJackTrip::appendPlugins` seems a little weird to me, but maybe it makes more sense to you?

Reply:
I guess. I just would like the separation to exist, I think. So that every JackTrip process does not necessarily have this plugin in the chain that it may or may not be using. Is there a way to instantiate it within the VS side of things? 

Maybe not. You'd know better.

Reply:
I do think having it there where they're instantiated with the GUI makes the most sense. We'll have to adapt it for VS mode because right now we don't use any plugins in that mode, but in this case, we just don't have any check to include the VU meter like we would for the others.

Reply:
I agree that it makes sense to move it out of the main JackTrip thread. The instantiation is now in `virtualstudio.cpp`. It took a bit of time because there was an issue related to how many channels are actually being metered (the issue specifically affected mono microphone inputs with RTAudio), but I found a workaround for that.

With this update, we can still implement VU meters with classic mode, though we would need to have separate instantiations that happen in `qjacktrip.cpp`.